src/app/components/sites/manage-page/template.ts

```diff
@@ -28,9 +28,15 @@ export default class AppSitesManagePageTemplate extends Vue {
 	}
 
 	enable() {
-		if (this.site) {
-			return this.site.$activate();
+		if (!this.site) {
+			return;
 		}
+
+		this.site.$activate().catch(e => {
```

Comment:
Don't we need to return the promise or resolve it? I think this code currently doesn't do anything.

Reply:
Nah, this works as intended. it's not like the html click handler awaits on the promise or anything.